### PR TITLE
[FEATURE] Gravatar viewhelper

### PR DESCRIPTION
**Pull request by jmverges**
Closes: #884

### CODE REVIEW

**Comment by cedricziel** [comment]
Small hint: You can put the "closes" in the second line or in the merge request only as well

**Comment by jmverges** [comment]
Thank you for the advices @cedricziel 
I just disagree with the int casting. By default gravatar provides 80px size, so with some default value in the parameter you will get an additional parameter not needed in the generated url.
I take note about everything else.

File: Tests/Unit/ViewHelpers/Media/GravatarViewHelperTest.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+/**
+ * @protection off
+ * @author Juan Manuel Vergés Solanas <juanmanuel@vergessolanas.es>
+ * @package Vhs
+ * @subpackage ViewHelpers\Media
+ */
+class GravatarViewHelperTest extends AbstractViewHelperTest {
+
+	protected $arguments = [
```
Comment (NamelessCoder) [inline Tests/Unit/ViewHelpers/Media/GravatarViewHelperTest.php]: Needs phpdoc, also in other test case.

File: Tests/Unit/ViewHelpers/Media/GravatarViewHelperTest.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		'email' => 'juanmanuel.vergessolanas@gmail.com',
+		'secure' => FALSE,
+	];
+
+	/**
+	 * @test
+	 */
+	public function compareResult() {
```
Comment (NamelessCoder) [inline Tests/Unit/ViewHelpers/Media/GravatarViewHelperTest.php]: Needs a human-readable and clearly labelling name, e.g. something like `generatesExpectedUriForEmailAddress`.

**Comment by NamelessCoder** [comment]
Overall: needs replacing of `[]` to `array()` syntax. We will make the change, but it hasn't been made yet ;)

File: Tests/Unit/ViewHelpers/Media/GravatarViewHelperTest.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<?php
+namespace FluidTYPO3\Vhs\ViewHelpers\Media;
```
Comment (NamelessCoder) [inline Tests/Unit/ViewHelpers/Media/GravatarViewHelperTest.php]: `FluidTYPO3\Vhs\Tests\Unit\ViewHelpers\Media`

File: Tests/Unit/ViewHelpers/Uri/GravatarViewHelperTest.php [hunk 4]
```diff
@@ -0,0 +1,45 @@
+<?php
+namespace FluidTYPO3\Vhs\ViewHelpers\Uri;
```
Comment (NamelessCoder) [inline Tests/Unit/ViewHelpers/Uri/GravatarViewHelperTest.php]: `namespace FluidTYPO3\Vhs\Tests\Unit\ViewHelpers\Uri`

**Comment by cedricziel** [comment]
Thx!

**Pull request merged**